web/components/sink-item.js

```diff
@@ -25,7 +25,7 @@ template.innerHTML = `
 	user-select: none;
 	cursor: pointer;
 	padding: 0.7em;
-	background-color: var(--background-color, white);
+	XXbackground-color: var(--background-color, white);
```

Comment:
Typo? Or what is `XXbackground`?

Reply:
disabled it... forgot to remove thanks :)